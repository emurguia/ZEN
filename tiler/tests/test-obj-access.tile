// Tests that object attributes acn be accessed

class Click {
   attr: string str;
   attr: int a;
   attr: string b; 
}

init {
   <Click> idk;
   coord point;

   tile(3, 3);

   idk = new Click("yes", 10, "21");
   sprint(idk.str);
   iprint(idk.a);
   sprint(idk.b);

   close();
}