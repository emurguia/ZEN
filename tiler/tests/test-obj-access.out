yes
10
21
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.